apps/cyberstorm-nextjs/app/communities/layout.tsx

```diff
@@ -1,19 +1,11 @@
 import "@thunderstore/cyberstorm-styles";
 import rootStyles from "../RootLayout.module.css";
-import { BreadCrumbs } from "@thunderstore/cyberstorm";
 import React from "react";
-import { PageHeader } from "@thunderstore/cyberstorm/src/components/Layout/BaseLayout/PageHeader/PageHeader";
 
 export default function CommunitiesLayout(props: React.PropsWithChildren) {
```

Comment:
This now looks like a generic layout that can be shared by all/most of the pages?

Reply:
Yeah, it kinda is. I'm still figuring out the NextJS layout structure. I'll refine the structure and "deduplicate" layouts as much as possible, after I've converted all the layouts to be under NextJS.